### DISCUSSION TITLE
Radeon ProRender for Maya 3.3.36

### DISCUSSION THREAD

**Discussion by bsavery** [GPUOpen-LibrariesAndSDKs/RadeonProRenderMayaPlugin#314] [Announcements]
Adds:

- New features for changing an objects shadow properties.
- Updated options for fog and atmosphere effects.
- Updated Cryptomatte AOVs

and [more!](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderMayaPlugin/blob/master/CHANGELOG.md#version-34)

<hr /><em>This discussion was created from the release <a href='https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderMayaPlugin/releases/tag/v3.3.36'>Radeon ProRender for Maya 3.3.36</a>.</em>

**Comment by tapanCL** [comment]
How to install RPR in Maya 2023?

**Reply by bsavery** [reply]
We will add a 2023 build soon!
